File: solutions/uri/1009/1009.r
input <- file('stdin', 'r')

a <- readLines(input, n=1)
b <- as.integer(readLines(input, n=1))
c <- as.double(readLines(input, n=1))

write(sprintf("TOTAL = R$ %.2f", b + (c * 0.15)), "")

File: solutions/uri/1009/1009.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')

const [, _b, _c] = input
  .trim()
  .split('\n')
const [b, c] = [_b, _c].map(x => parseFloat(x))

console.log(`TOTAL = R$ ${(b + (c * 0.15)).toFixed(2)}`)
